yt_dlp/extractor/funker530.py

```diff
@@ -0,0 +1,75 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from .rumble import RumbleEmbedIE
+from .youtube import YoutubeIE
+from ..utils import clean_html
+
+
+class Funker530IE(InfoExtractor):
+    _VALID_URL = r'https?:\/\/(?:www\.)?funker530\.com\/video\/(?P<id>[^\/]+)\/?'
```

Comment:
don't need to escape forward slashes, and also watch out for queries and fragments at the end of the `id`
```suggestion
    _VALID_URL = r'https?://(?:www\.)?funker530\.com/video/(?P<id>[^/?#]+)'
```